lib/routes/whu/news.ts

```diff
@@ -13,15 +13,40 @@ import { domain, processMeta, getMeta, processItems } from './util';
 
 export const route: Route = {
     path: '/news/:category{.+}?',
-    name: 'Unknown',
+    categories: ['university'],
+    example: '/whu/news',
+    parameters: { category: '新闻栏目，可选' },
+    name: '武汉大学新闻',
     maintainers: [],
     handler,
+    description: `
+    category 参数可选，范围如下:
+
+    | 新闻栏目 | 武大资讯 | 学术动态 | 珞珈影像 | 武大视频 |
+    | -------- | -------- | -------- | -------- | -------- |
+    | 参数     |  0 或 \`wdzx/wdyw\`  | 1 或 \`kydt\` | 2 或 \`stkj/ljyx\` | 3 或 \`stkj/wdsp\` |
+
+    另外 route 后可以加上 \`?limit=n\` 的查询参数，表示只获取前 n 条新闻；如果不指定默认为 10。
+    `,
```

Comment:
```suggestion
category 参数可选，范围如下:

| 新闻栏目 | 武大资讯 | 学术动态 | 珞珈影像 | 武大视频 |
| -------- | -------- | -------- | -------- | -------- |
| 参数     |  0 或 \`wdzx/wdyw\`  | 1 或 \`kydt\` | 2 或 \`stkj/ljyx\` | 3 或 \`stkj/wdsp\` |

另外 route 后可以加上 \`?limit=n\` 的查询参数，表示只获取前 n 条新闻；如果不指定默认为 10。`,
```
Indent MD content with 0 or 2 spaces, otherwise they won't be rendered properly